packages/integrations/vercel/test/redirects.test.js

```diff
@@ -32,23 +34,41 @@ describe('Redirects', () => {
 	it('define static routes', async () => {
 		const config = await getConfig();
 
-		const oneRoute = config.routes.find((r) => r.src === '/\\/one');
+		const oneRoute = config.routes.find((r) => r.src === '/one');
 		expect(oneRoute.headers.Location).to.equal('/');
 		expect(oneRoute.status).to.equal(301);
 
-		const twoRoute = config.routes.find((r) => r.src === '/\\/two');
+		const twoRoute = config.routes.find((r) => r.src === '/two');
 		expect(twoRoute.headers.Location).to.equal('/');
 		expect(twoRoute.status).to.equal(301);
 
-		const threeRoute = config.routes.find((r) => r.src === '/\\/three');
+		const threeRoute = config.routes.find((r) => r.src === '/three');
 		expect(threeRoute.headers.Location).to.equal('/');
 		expect(threeRoute.status).to.equal(302);
 	});
 
+	it('define external routes', async () => {
+		const config = await getConfig();
+
+		const externalRoute = config.routes.find((r) => r.src === '/external');
+		expect(externalRoute).to.not.be.undefined;
+		expect(externalRoute.headers.Location).to.equal('https://example.com/');
+		expect(externalRoute.status).to.equal(301);
+	});
```

Comment:
As discussed in `adapters` repo, this shouldn't be a valid case. We don't support external redirects... right?